packages/client/types/constEnums.ts

```diff
@@ -391,9 +391,9 @@ export const enum ZIndex {
   SIDEBAR = 16,
   SIDE_SHEET = 8,
   SNACKBAR = 24 /* snackbar is kind of dialog */,
-  SPOTLIGHT_MODAL = 25, // above the tooltip
   REFLECTION_IN_FLIGHT = 8,
-  REFLECTION_IN_FLIGHT_LOCAL = 26, // keep it above the dialog
+  REFLECTION_IN_FLIGHT_LOCAL = 28, // keep it above the dialog
+  REFLECTION_IN_FLIGHT_SPOTLIGHT = 26,
```

Comment:
Remote reflections that are in the Spotlight (but not those that aren't in the Spotlight) need to appear above the Spotlight modal but below local reflections. The Spotlight modal uses usePortal which uses the dialog zIndex of 24. usePortal is also used by expandedReflectionsStack and a bunch of other modals which, unlike the Spotlight, need to appear above the remote reflections.

I think the cleanest approach is adding a `REFLECTION_IN_FLIGHT_SPOTLIGHT` zIndex and then passing an `isInSpotlight` prop into `useDraggableReflectionCard`